src/map/map.css

```diff
@@ -23,9 +23,10 @@
   background-image: url('MapsGuy.png');
   background-repeat: no-repeat;
   background-size: contain;
-  background-position-x: left 8px;
-  width: 30px;
-  height: 30px;
+  background-position-x: left 6px;
```

Comment:
I'd recommend looking into this approach using offsets when initialising the current marker: https://docs.mapbox.com/mapbox-gl-js/api/markers/

This approach looks like it worked for x but we really want the bear to be standing where the position is instead of being placed centered onto the position.